src/main/java/org/jboss/set/cryo/process/ExecuteProcess.java

```diff
@@ -76,6 +85,7 @@ public OperationResult getProcessResult() {
             result = process.waitFor();
             output = readOutput(process);
             }
+            fWriter.close();
```

Comment:
You should move this to the `finally` block to ensure there's no leak in case of an exception

Reply:
Sure